### PR TITLE
feat: add twilio and add payment card

### PR DESCRIPTION
**Pull request by devdhruti**

### CODE REVIEW

File: app/controllers/meetings_controller.rb [hunk 1]
```diff
@@ -2,7 +2,12 @@ class MeetingsController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @meetings = current_user.meetings.all.order('start_time')
+    if current_user.admin?
+      meeting = Meeting.all
+    else
+      meeting = current_user.meetings
+    end
+    @meetings = meeting.all.order('start_time')
```
Comment (neshmapatel-simformsolutions) [inline app/controllers/meetings_controller.rb]: meeting.order would work, no need for meeting.all.order
check once though

**Review by neshmapatel-simformsolutions — CHANGES_REQUESTED** [review]

File: app/controllers/meetings_controller.rb [hunk 2]
```diff
@@ -2,7 +2,12 @@ class MeetingsController < ApplicationController
...
     @todays_meeting = @meetings.select {|x| x.start_time.day == Time.current.day}
     @upcoming_meeting = @meetings.select {|x| x.start_time.day > Time.current.day}
     @completed_meeting = @meetings.select {|x| x.start_time.day < Time.current.day}
```
Comment (neshmapatel-simformsolutions) [inline app/controllers/meetings_controller.rb:13]: Time.current.day instead
DateTime.today may also work.

File: app/models/meeting.rb [hunk 3]
```diff
@@ -6,6 +6,8 @@ class Meeting < ApplicationRecord
   validate :end_after_start
   validates :start_time, :end_time, :presence => true, :overlap => {:scope => "user_id"}
 
+  scope :tomorrow_meeting, -> { where(start_time: Date.tomorrow.beginning_of_day .. Date.tomorrow.end_of_day) }
```
Comment (neshmapatel-simformsolutions) [inline app/models/meeting.rb]: as it may have many meetings, lets make it plural,
tomorrow_meetings

File: app/jobs/send_message.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+class SendMessage
+  def perform
+    if Meeting.tomorrow_meeting.present?
```
Comment (neshmapatel-simformsolutions) [inline app/jobs/send_message.rb]: only Meeting.tomorrow_meeting may also work, please check though.

File: app/jobs/send_message.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+      Meeting.tomorrow_meeting.each do |meeting|
+        message = "Hey, #{meeting.user.email}, we’re eagerly waiting to see to at the meeting #{meeting.title}, tomorrow at #{meeting.start_time.strftime("%I:%M%p")}"
```
Comment (neshmapatel-simformsolutions) [inline app/jobs/send_message.rb:6]: we need to send name of the user and not email, meeting.user.email

File: app/models/user.rb [hunk 6]
```diff
@@ -1,9 +1,16 @@
...
+
+  validates :phone, phone: true
 
   # Include default devise modules. Others available are:
   # :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+
+  def admin?
+    AdminUser.find_by(email: self.email).present? ? true : false
```
Comment (neshmapatel-simformsolutions) [inline app/models/user.rb]: AdminUser.find_by(email: self.email).present? 
this is sufficient to return true or false

File: app/views/devise/registrations/edit.html.erb [hunk 7]
```diff
@@ -31,6 +31,11 @@
     <%= f.password_field :current_password, autocomplete: "current-password" %>
   </div>
 
+  <div class="field">
+    <%= f.label :phone_number %><br />
+    <%= f.phone_field :phone, autocomplete: "phone" , placeholder: "+(91)123456789"%>
```
Comment (neshmapatel-simformsolutions) [inline app/views/devise/registrations/edit.html.erb]: while updating, no need to give placeholder for the phone number, you can display user's phone number if you want to.

File: lib/tasks/meeting.rake [hunk 8]
```diff
@@ -0,0 +1,6 @@
+namespace :meeting do
+  desc "TODO"
```
Comment (neshmapatel-simformsolutions) [inline lib/tasks/meeting.rake]: change description based on what task is supposed to perform.

File: log/cron.log [hunk 9]
```diff
@@ -0,0 +1 @@
+warning ../../package.json: No license field
```
Comment (neshmapatel-simformsolutions) [inline log/cron.log:1]: need to add log files in gitignore.

File: app/controllers/meetings_controller.rb (hunk 2 above)
Comment (devdhruti) [reply app/controllers/meetings_controller.rb:13]: DateTime.today is not working instead DateTime.current.day is working

File: app/jobs/send_message.rb (hunk 4 above)
Comment (devdhruti) [reply app/jobs/send_message.rb]: not working

File: app/jobs/send_message.rb (hunk 5 above)
Comment (devdhruti) [reply app/jobs/send_message.rb:6]: I am fetching only mail from user so I wrote meeting.user.email

Comment (neshmapatel-simformsolutions) [reply app/jobs/send_message.rb:6]: okay

File: app/services/twilio_client.rb [hunk 10]
```diff
@@ -6,6 +6,7 @@ def initialize
   end
 
   def send_text(user, message)
+    binding.pry
```
Comment (neshmapatel-simformsolutions) [inline app/services/twilio_client.rb]: need to remove this.

**Review by neshmapatel-simformsolutions — APPROVED** [review]

**Pull request merged**